src/main/webapp/web.xml

```diff
@@ -7,6 +7,46 @@
         <servlet-name>index</servlet-name>
         <servlet-class>mate.controller.IndexController</servlet-class>
     </servlet>
+    <servlet>
+        <servlet-name>getAllUsersController</servlet-name>
+        <servlet-class>mate.controller.DisplayDriversController</servlet-class>
+    </servlet>
+    <servlet>
+        <servlet-name>createDriverController</servlet-name>
+        <servlet-class>mate.controller.CreateDriverController</servlet-class>
+    </servlet>
+    <servlet>
+        <servlet-name>createCarController</servlet-name>
+        <servlet-class>mate.controller.CreateCarController</servlet-class>
+    </servlet>
+    <servlet>
+        <servlet-name>createNewManufacturerController</servlet-name>
+        <servlet-class>mate.controller.CreateManufacturerController</servlet-class>
+    </servlet>
+    <servlet>
+        <servlet-name>addDriverToCarController</servlet-name>
+        <servlet-class>mate.controller.AddDriverToCarController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>getAllUsersController</servlet-name>
+        <url-pattern>/driver/all</url-pattern>
+    </servlet-mapping>
+    <servlet-mapping>
+        <servlet-name>createDriverController</servlet-name>
+        <url-pattern>/driver/create</url-pattern>
+    </servlet-mapping>
+    <servlet-mapping>
+        <servlet-name>addDriverToCarController</servlet-name>
+        <url-pattern>/car/add</url-pattern>
+    </servlet-mapping>
+    <servlet-mapping>
+        <servlet-name>createCarController</servlet-name>
+        <url-pattern>/car/create</url-pattern>
+    </servlet-mapping>
+    <servlet-mapping>
+        <servlet-name>createNewManufacturerController</servlet-name>
+        <url-pattern>/manufacturer/create</url-pattern>
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-web/web-practice/java-web-practice): URLs naming in project